### PR TITLE
Feature onboarding pages

### PR DESCRIPTION
**Pull request by freddy2659**
Still more work to go on the profile_settings page. The other two pages need doing as well.

### CODE REVIEW

**Review by jassalss — APPROVED** [review]
looks fine. Very happy to see that you are using css via material ui.

File: client/src/component/ProfileWidget.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+                            <Typography align="left" variant="h6">
+                                {props.heading}
+                            </Typography>
+                        </Grid>
+                        <Grid item xs>
+                            <ProgressBar value={20} />
+                        </Grid>
+                    </Grid>
+                </Box>
+                <Divider/>
```
Comment (sina-jamshidi) [inline client/src/component/ProfileWidget.js]: great use of MUI components and styling!

**Review by sina-jamshidi — COMMENTED** [review]
Looks good so far

**Review by jassalss — COMMENTED** [review]
Looks very nice.

**Comment by freddy2659** [comment]
Technically not quite finished, but close enough for the demo

**Pull request merged**